connection.php

```diff
@@ -0,0 +1,8 @@
+<?php
+//$db = new mysqli('localhost:8889', 'root', 'root', 'attendance_new');
+	$db = new mysqli('sql3.freemysqlhosting.net:3306', 'sql3218514', 'fxBw214YQ4', 'sql3218514');
```

Comment:
You might not want this in a public repo 🤔

Reply:
Geez, I'm catching hell for this in Orcas right now too. Who cares for now? It's just for testing.

Reply:
Better than everyone swapping DB files to their local AMPs

Reply:
Well I guess it's fine as long as you have a backup. People scrape github looking for this kinda thing so your tables could probably get dropped.

Reply:
Swapping dbs around is bad, but you could just use a gitignore to remove the connection file.

Reply:
Oh Milo... 😍

Reply:
Jack!!!!!!!!

Reply:
A comment